scripts/llbc_extract.py

```
@@ -651,7 +610,147 @@ def refuse_nested_repos(entries: list[str]) -> None:
     raise SystemExit("\n".join(lines))
 
 
-_inputs_cache: dict[tuple, tuple[list[str], list[Path]]] = {}
+_inputs_cache: dict[tuple, tuple[list[Path], list[Path]]] = {}
+
+
+_FILES_KEY = b'"files":['
+
+
+def ullbc_files_table(path: Path, chunk_size: int = 1 << 22) -> list[dict]:
```

Ruff (0.16.1)

[warning] Too many branches (15 > 12)

(PLR0912)